HSF-Notes/2018-01/espp-hsf-submission.tex

```diff
@@ -0,0 +1,168 @@
+% This is the styling for submission to arXiv using JHEP Preprint Style
+
+% Copyright (C) 2018, the authors, licence CC-BY-4.0.
+
+% JHEP preprint template
+\documentclass[11pt,a4paper]{article}
+\usepackage{jheppub}
+\notoc
+
+\usepackage{graphicx}
+\usepackage{subcaption}
+%\usepackage[titletoc]{appendix}
+\usepackage{xspace}
+
+\usepackage[utf8]{inputenc}
+
+%% Comment out before final submission
+%\usepackage{lineno}  % for line numbering during review
+%\linenumbers
+
+\abstract{In 2017 the experimental High-Energy Physics community wrote a
+\emph{Roadmap for HEP Software and Computing R\&D for the
+2020s}\footnote{\href{https://arxiv.org/abs/1712.06982}{\emph{arXiv:1712.06982}}}.
+This effort was organised by the HEP Software
+Foundation\footnote{\href{https://hepsoftwarefoundation.org/}{\emph{https://hepsoftwarefoundation.org/}}}
+(HSF) and was supported by more than 300 physicists from more than 100
+institutes worldwide. It delivered a strategy outlining the most
+important areas in which investment is needed to ensure the success of
+our experimental programme. This contribution to the ESPP is an
```

Comment:
Apologies for being a pain - we could spell out ESPP in the abstract at least.

Reply:
Well, the problem is now that we made the submission, so I don't want the archived copy of the document to differ from what we actually gave in to the process.